packages/eas-cli/src/commands/release/create.ts

```diff
@@ -0,0 +1,114 @@
+import { getConfig } from '@expo/config';
+import { Command, flags } from '@oclif/command';
+import chalk from 'chalk';
+import branchName from 'current-git-branch';
```

Comment:
The `current-git-branch` package has way too many dependencies, we should do something lighter.
```
info Direct dependencies
└─ current-git-branch@1.1.0
info All dependencies
├─ babel-plugin-add-module-exports@0.2.1
├─ cross-spawn@5.1.0
├─ current-git-branch@1.1.0
├─ get-stream@3.0.0
├─ is-git-repository@1.1.1
├─ is-stream@1.1.0
├─ isexe@2.0.0
├─ lru-cache@4.1.5
├─ npm-run-path@2.0.2
├─ p-finally@1.0.0
├─ path-is-absolute@1.0.1
├─ path-key@2.0.1
├─ pseudomap@1.0.2
├─ shebang-command@1.2.0
├─ shebang-regex@1.0.0
├─ signal-exit@3.0.3
├─ strip-eof@1.0.0
├─ which@1.3.1
└─ yallist@2.1.2
```
Maybe something like `spawnSync('git', ['rev-parse', '--abbrev-ref', 'HEAD'])` (and catch the error if it fails).

Reply:
great point.

Reply:
May need to use `@expo/spawn-async` (or `cross-spawn`) for Windows support, not sure

Reply:
I used `@expo/spawn-async`

Reply:
(added to the utils/git file).